src/app/home-page/state/recent-events.service.ts

```diff
@@ -1,17 +1,24 @@
 import { Injectable } from '@angular/core';
 import { ID } from '@datorama/akita';
-import { Event, EventsService } from 'app/shared/openapi';
+import { Event, EventsService, User } from 'app/shared/openapi';
+import { UsersService } from 'app/shared/swagger';
+import { UserQuery } from 'app/shared/user/user.query';
 import { finalize, tap } from 'rxjs/operators';
 import { RecentEventsStore } from './recent-events.store';
 
 @Injectable({ providedIn: 'root' })
 export class RecentEventsService {
-  constructor(private recentEventsStore: RecentEventsStore, private eventsService: EventsService) {}
+  constructor(
+    private recentEventsStore: RecentEventsStore,
+    private eventsService: EventsService,
+    protected userQuery: UserQuery,
```

Comment:
It doesn't look like you're using `userQuery` nor `usersService`.

I tried removing them locally and got some weird compilation errors, but I think you should figure out how to remove them (unless I'm wrong about them being used).

Reply:
> I'm using `userQuery` and `usersService` to find the user either from the logged in user or from the username respectively.

I may have my IDE code configured incorrectly -- what lines of code are they invoked from?

Reply:
Oh wait nevermind, I was looking at the wrong file. I removed them locally on mine and didn't come across any compilation errors